pymongo/logger.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
+
+import enum
+import os
+from typing import Any
+
+from bson import UuidRepresentation, json_util
+from bson.json_util import JSONOptions
+
+
+class LogMessageStatus(str, enum.Enum):
```

Comment:
Let's prefix all globals in this file with `_` to avoid people mistakenly thinking this is public.